### PR TITLE
add check on process.env.HOST in index.ts.ejs file

### PR DESCRIPTION
**Pull request by atefBB**
`process.env.HOST` may be `undefined` so we need to add a check on it !

## Checklist

- [ ] DCO (Developer Certificate of Origin) [signed in all commits](https://loopback.io/doc/en/contrib/code-contrib.html)
- [ ] `npm test` passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the [style guide](http://loopback.io/doc/en/contrib/style-guide.html)
- [ ] API Documentation in code was updated
- [ ] Documentation in [/docs/site](../tree/master/docs/site) was updated
- [ ] Affected artifact templates in `packages/cli` were updated
- [ ] Affected example projects in `examples/*` were updated

👉 [Check out how to submit a PR](https://loopback.io/doc/en/lb4/submitting_a_pr.html) 👈

### CODE REVIEW

**Review by dhmlau — COMMENTED** [review]
@atefBB, thanks for the PR. I just have one minor comment. Thanks.

File: packages/cli/generators/app/templates/src/index.ts.ejs [hunk 1]
```diff
@@ -19,7 +19,7 @@ if (require.main === module) {
   const config = {
     rest: {
       port: +(process.env.PORT ?? 3000),
-      host: process.env.HOST,
+      host: process.env.HOST !== undefined ? process.env.HOST : "127.0.0.1",
```
[the remark this answers is not in the document]
Comment (atefBB) [reply packages/cli/generators/app/templates/src/index.ts.ejs]: no prob !

**Comment by atefBB** [comment]
DONE @dhmlau !

**Comment by dhmlau** [comment]
Thanks @atefBB. There seems to be some linting issue with the commit messages. 
```
⧗   input: Merge bb408e97191bd9d441dd4a9392c87036df23554c into b0e4a869679b8bb4ca2c48c233348d582dd4752a


✔   found 0 problems, 0 warnings
⧗   input: Update packages/cli/generators/app/templates/src/index.ts.ejs

Co-authored-by: Diana Lau <dhmlau@ca.ibm.com>
Signed-off-by: Atef Ben Ali <atefBB@users.noreply.github.com>
✖   subject may not be empty [subject-empty]
✖   type may not be empty [type-empty]

✖   found 2 problems, 0 warnings
ⓘ   Get help: https://github.com/conventional-changelog/commitlint/#what-is-commitlint

⧗   input: Update index.ts.ejs

`process.env.HOST` may be `undefined` so we need to add a check on it !

Signed-off-by: Atef Ben Ali <atefBB@users.noreply.github.com>
✖   subject may not be empty [subject-empty]
✖   type may not be empty [type-empty]

✖   found 2 problems, 0 warnings
ⓘ   Get help: https://github.com/conventional-changelog/commitlint/#what-is-commitlint

Error: Process completed with exit code 1.
```

Could you please squash the commits? The commit message could be something like: 

```
fix(cli): add check on process.env.HOST
```

Please see [our commit guideline](https://loopback.io/doc/en/lb4/code-contrib-lb4.html#commit-message-guidelines) for details. Thanks.

**Comment by atefBB** [comment]
ok ! I'll do !

**Comment by atefBB** [comment]
I think it's DONE @dhmlau !

**Review by dhmlau — APPROVED** [review]

**Comment by dhmlau** [comment]
Thanks @atefBB. Your PR has been merged! 🎉

**Comment by atefBB** [comment]
thx @dhmlau

**Pull request merged**